Libraries/LibWeb/HTML/HTMLInputElement.cpp

```diff
@@ -59,6 +59,24 @@
 #include <LibWeb/WebIDL/DOMException.h>
 #include <LibWeb/WebIDL/ExceptionOr.h>
 
+namespace {
+
+// In the longterm we might want to replace doubles with a type that has a higher precision (e.g. a manual decimal implementation).
```

Comment:
Do we know of cases where this actually is needed to support the web? If so, consider making this a `// FIXME` or create a GitHub issue with a test case. Otherwise, I'd leave this comment out.

Reply:
I'm not entirely sure, I just had this thought because Firefox does it this way and can therefore handle extremely small/large numbers. I'm not sure, but the spec only requires 32-bit float, right? In that case, it wouldn't be necessary, and I think inaccuracies in the calculation would be in line with the specification.

Reply:
In that case I'd remove this comment from the code, and if you find some cases where it's actually necessary to have more precision we can always raise an issue / create a new PR. No need to keep track of things that seem to work just fine imo :)